pages/linux/fdisk.md

```diff
@@ -10,3 +10,27 @@
 - Start the partition manipulator:
 
 `fdisk {{/dev/sdX}}`
```

Comment:
```suggestion
`sudo fdisk {{/dev/sdX}}`
```

Reply:
Any thoughts on this, since it is using block devices it needs root to do anything really

Reply:
From the guidline:
> When in doubt, keep new command-line users in mind. Err on the side of clarity rather than terseness. For example, commands that require `sudo` should include it directly in the examples.

Reply:
`fdisk: cannot open /dev/sda: Permission denied`


Block devices have root permissions attached, so I will add it.